src/wishlist.py

```diff
@@ -39,14 +40,15 @@ def wishlist():
 '''
 @wishlist_bp.route('/wishlistAdd/', methods=['POST'])
 def wishlistAdd():
-    user = request.json['user']
-    item = request.json['item']
+    data = request.get_json(force=True)
+    user = data.get('user')
+    item = json.loads(data.get('item'))
 
     # TODO convert price to generic currency and fix price hack used below
 
     db.query(
         'INSERT INTO wishlist (username, name, price, website, link, rating) VALUES(%s, %s, %s, %s, %s, %s)',
-        (user, item['title'], float(item['price'][1:]), item['website'], item['link'], item['rating'])
+        (user, item['title'], float(item['price']), item['website'], item['link'], item['rating'])
```

Comment:
Right now the price is sent as a string including the $ sign. Could you look into making it store a 'universal' currency in the database and convert to appropriate currency when being sent to frontend and when taken from frontend?

Reply:
Assumed taking a number only for price, cut the inputs in the endpoint call down to a float from frontend would work as well, all of the tests are setup already for not sending in a $ so probably an quicker fix just to not send it